CHANGELOG.asciidoc

```diff
@@ -114,6 +114,8 @@ See the <<upgrade-to-v4>> guide.
 [float]
 ===== Features
 
+* Add support for `@aws-sdk/client-dynamodb`, one of the AWS SDK v3 clients.
+  ({issues}2958[#2958])
```

Comment:
This should move up to the `Unreleased` block.

Reply:
done!